controller/templates/confd_settings.py

```diff
@@ -10,6 +10,9 @@
 except:
     SCHEDULER_OPTIONS = {}
 
+# scheduler swarm manager host
+SWARM_HOST = """{{ or (.deis_scheduler_swarm_host) "" }}"""
```

Comment:
I believe we should be using the existing config option SCHEDULER_TARGET for this